lib/perl/Genome/InstrumentData/AlignmentResult/Merged.pm

```diff
@@ -333,11 +333,11 @@ sub _remove_per_lane_bam {
 }
 
 sub _get_temp_allocation {
-    my ($self, $alignment) = @_;
+    my ($self, $alignment_id, $output_dir) = @_;
     return Genome::Disk::Allocation->create(
         disk_group_name     => $ENV{GENOME_DISK_GROUP_ALIGNMENTS},
-        allocation_path     => 'merged/recreated_per_lane_bam/'.$alignment->id.'_'._get_uuid_string(),
-        kilobytes_requested => Genome::Sys->disk_usage_for_path($self->output_dir),
+        allocation_path     => 'merged/recreated_per_lane_bam/'.$alignment_id.'_'._get_uuid_string(),
+        kilobytes_requested => Genome::Sys->disk_usage_for_path($output_dir),
```

Comment:
How does the `$output_dir` passed in by the HtSeq Count result relate to the size of the per-lane BAM that needs to be stored in this allocation?